ui/packages/consul-ui/app/router.js

```diff
@@ -199,6 +199,9 @@ export const routes = {
           'binding-rules': {
             _options: { path: '/binding-rules' },
           },
+          'namespace-rules': {
+            _options: { path: '/namespace-rules' },
```

Comment:
Bit of a quibble this one. The URL here is great, although we refer to any Routes that are namespace related as `nspace`. So maybe we should call this one `nspace-rules`. 

I'm trying to imagine myself writing this in amongst some code and wondering which one I would write automatically without thinking `nspace-rules` or `namespace-rules` - as I've trained myself in this codebase to never write the word `namespace` I'd probably go with `nspace-rules`.

Reply:
Renamed to be `nspace-rules`